client/stylesheets/_toCat.scss

```diff
@@ -4796,3 +4798,10 @@ On click the images will be displayed at normal size to complete the effect
 .sp-palette-row-2 {
     display: none !important;
 }
+
+
+@media (min-width: 1200px) {
+    .oneDiv, .onediv {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `oneDiv` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line